Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 [dependencies]
 futures = "0.3.26"
 hex = "0.4"
-libp2p = { version = "0.51.0", features = [ "identify", "macros", "ping", "tokio" ]}
+libp2p = { version = "0.51.0", features = [ "identify", "macros", "ping", "tokio", "tcp", "dns", "websocket", "noise", "mplex", "yamux" ]}
```

Comment:
is the development transport using tcp or websockets? from the docs it says tcp, but seems weird we need websockets too